tools/unreal-vat-demo/Content/Python/build_vat_demo.py

```
@@ -0,0 +1,453 @@
+    if not verify_gltf_has_uv2():
+        unreal.log_warning(
+            "=== Bootstrap STOPPED: source.gltf is missing TEXCOORD_1. "
+            "Re-bake with `qtmesh vat <file> --anim <name> --emit-uv2 "
+            "-o <dir>` and overwrite the files under Content/Rumba/. ===")
+        return
+    frames, mn, mx = read_sidecar()
+    unreal.log("Sidecar: frames=%d, min=%s, max=%s" % (frames, mn, mx))
+    build_material(frames, mn, mx)
+    spawn_dancer_in_level()
+    unreal.log("=== Bootstrap done. Look for OpenVAT_Dancer in the "
+               "viewport — material drives the animation via Time × fps, "
```

Ruff (0.15.13)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)